build/Dockerfile.base

```
@@ -78,13 +79,41 @@ RUN groupadd --gid 53150 -r easy_infra \
  && rm -f /usr/local/bin/envconsul.zip \
  && chmod 0755 /usr/local/bin/envconsul \
  && apt-get remove -y unzip \
- && apt-get install -y --no-install-recommends cmake build-essential flex bison libyaml-dev libssl-dev \
+ && apt-get install -y --no-install-recommends cmake \
+                                               build-essential \
+                                               flex \
+                                               bison \
+                                               libyaml-dev \
+                                               libz-dev \
+                                               libssl-dev \
+                                               libcurl4-gnutls-dev \
+                                               libexpat1-dev \
+                                               make \
+                                               gcc \
+ # Build a new version of the git binary to support loading environment variables to designate safe directories
+ # The minimum required version to support this method is 2.38.0 as per https://github.com/git/git/commit/6061601d9f1f1c95da5f9304c319218f7cc3ec75
+ && cd /tmp \
+ && mkdir git \
+ && curl -L https://mirrors.edge.kernel.org/pub/software/scm/git/git-${GIT_VERSION#v}.tar.gz -o git.tar.gz \
+ && tar -zxf git.tar.gz -C /tmp/git --strip-components=1 \
+ && cd git \
```

nit: make absolute and handle in one step

```suggestion
 && cd /tmp/git \
```